src/commands/export.ts

```diff
@@ -1,10 +1,12 @@
 import { CommandContext } from "../command_handler.ts";
 import { EnvChange, ExecuteResult } from "../result.ts";
 
+// All commands needs to have async defintion
+// deno-lint-ignore require-await
```

Comment:
Again, we could remove this and allow synchronous commands.

Reply:
So all commands should be comes sync ? or is there a way to make it both sync and async ?
Current signature https://github.com/dsherret/dax/blob/main/src/command_handler.ts#L23

Reply:
Actually you could just use Promise.resolve (can’t respond much atm)

Reply:
Don't worry about it, when you have time feel free to review it

I went with making commandhandler ` Promise<ExecuteResult> |  ExecuteResult` and when it gets called I just await (I can branch on its type promise or not)